coalib/output/ConsoleInteraction.py

```diff
@@ -609,6 +609,51 @@ def print_actions(console_printer, section, actions, failed_actions):
     return get_action_info(section, actions[choice - 1], failed_actions)
 
 
+def apply_action_to_section(action,
+                            section,
+                            action_name,
+                            result,
+                            file_dict,
+                            file_diff_dict,
+                            console_printer,
+                            failed_actions):
+    """
+    Applies the given action.
+
+    :param action:          The action to the applied.
+    :param section:         Currently active section.
+    :param action_name:     The name of the action.
+    :param result:          Result corresponding to the actions.
+    :param file_diff_dict:  If it is an action which applies a patch, this
+                            contains the diff of the patch to be applied to
+                            the file with filename as keys.
+    :param file_dict:       Dictionary with filename as keys and its contents
+                            as values.
+    :param console_printer: Object to print messages on the console.
```

Comment:
@Makman2 I've confusion between `log_printer` and `console_printer`, from the codebase and currently filed issues it looks like `log_printer` is deprecated. Here I'm using `console_printer` to print the success message after applying the action, and it also involves formatting and colors. Do you want me to use `logging.info` instead to display the success message and use `assertLogs` in the tests?